server/checks/Approval.js

```diff
@@ -162,16 +245,19 @@ export default class Approval extends Check {
           debug(`${repository.full_name}#${issue.number}: Ignoring comment, not a PR`)
           return
         }
-
         sha = pr.head.sha
-
         // set status to pending first
         await github.setCommitStatus(user, repoName, sha, pendingPayload, token)
-
-        let dbPR = await this.getOrCreateDbPullRequest(pullRequestHandler, dbRepoId, issue.number);
-
-        await this.countApprovalsAndSetCommitStatus(config, user, repository, issue.number, dbPR.last_push, token, github, minimum, sha)
-        info(`${repository.full_name}#${issue.number}: Comment added`)
+        // read last push date from db
+        let dbPR = await pullRequestHandler.onGet(dbRepoId, issue.number)
```

Comment:
Yes, probably didn't pay attention during merge.

On 18 May 2016 at 08:52, Daniel Spaude <notifications@github.com> wrote:

> In server/checks/Approval.js
> <https://github.com/zalando/zappr/pull/289#discussion_r63653618>:
>
> >          // set status to pending first
> >          await github.setCommitStatus(user, repoName, sha, pendingPayload, token)
> > -
> > -        let dbPR = await this.getOrCreateDbPullRequest(pullRequestHandler, dbRepoId, issue.number);
> > -
> > -        await this.countApprovalsAndSetCommitStatus(config, user, repository, issue.number, dbPR.last_push, token, github, minimum, sha)
> > -        info(`${repository.full_name}#${issue.number}: Comment added`)
> > +        // read last push date from db
> > +        let dbPR = await pullRequestHandler.onGet(dbRepoId, issue.number)
>
> u can use getOrCreateDbPullRequest here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando/zappr/pull/289/files/3814c2c860c37d9405c145ccd1170c6a02342d83#r63653618>
>



-- 

undefined = () => {}

     __
   .´ o)=-
  /.-.'
 //  |\
 ||  |'  Nikolaus Piccolotto
,:)_/_   engineer.js